mantidproject/mantid
# Dev doc clean-up

**Description of work.**

Minor changes to remove warnings and errors from the dev-docs

- Corrected incorrect underline length
- Corrected duplicate link text
- Added newlines to fix rendering of horizontal sepparators, code block, image caption
- Removed old Muon testing doc (after consulting @AnthonyLim23)
- Corrected duplicate refference names
- Removed links to missing files

**To test:**

- Remove `build/dev-docs`
- Rebuild the `dev-docs-html` target
- Check the output for the number of warnings and for what they are
- Check the altered files display and opperate correctly

Fixes #34443


*This does not require release notes* because it is a maintenance change




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
